content/techniques/queues.md

```diff
@@ -194,6 +194,27 @@ const job = await this.audioQueue.add(
 );
 ```
 
+#### Request-scoped job processors
+
+Whether the processor needs to depend on a request scoped provider or be explicitly registered as a scoped processor, as follows:
+
+```typescript
+@Processor({
+  name: 'audio',
+  scope: Scope.REQUEST,
+})
+```
+
+then you can inject the `JOB_REF` via the constructor.
```

Comment:
```suggestion
Since request-scoped consumer classes are instantiated dynamically and scoped to a single job, you can inject a `JOB_REF` through the constructor using a standard approach.
```